common/src/main/java/com/wynntils/core/features/FeatureManager.java

```diff
@@ -273,7 +263,7 @@ private void registerFeature(Feature feature) {
     private void initializeFeature(Feature feature) {
         Class<? extends Feature> featureClass = feature.getClass();
 
-        // instance field
+        // Instance field
         try {
             Field instanceField = FieldUtils.getDeclaredField(featureClass, "INSTANCE", true);
```

Comment:
Just a note: I'd like to get rid of this in the future. If we really *do* need to get access to a Feature object, we should be able to get it through something like `Managers.Features.getInstance(SoulPointTimerFeature.class)`. But hopefully we can get rid of accessing feature instances altogether if we fix our config/storage story.

Reply:
Agreed.